ogr/ogrsf_frmts/oci/ogrocidatasource.cpp

```diff
@@ -838,6 +838,26 @@ OGRSpatialReference *OGROCIDataSource::FetchSRS(int nId)
         return nullptr;
     }
 
+    /* -------------------------------------------------------------------- */
+    /*      If the WKTEXT definition is equivalent to the EPSG SRS of code  */
+    /*      nId, return that EPSG SRS.                                      */
+    /* -------------------------------------------------------------------- */
+    if (nId < 32767 && papszResult[1] != nullptr && atoi(papszResult[1]) == nId)
```

Comment:
Can we use std::numeric_limits instead of the literal?

Reply:
> Can we use std::numeric_limits instead of the literal?

The 32767 indeed probably comes from the highest signed int16, but this is more than it is documented by EPSG as the (currently) highest number for their codes: https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset
I'll change the hard coded value to a HIGHEST_EPSG_CRS_NUMBER constant